.github/pull_request_template.md

```diff
@@ -58,22 +59,22 @@ Resolves #00
 - [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
 - [ ] Add at least 1 designer as PR reviewer
 
-### As code reviewer(s), I have
+### As a code reviewer, I have
 
 #### Reviewed, tested, and left feedback about the changes
 
 - [ ] Pulled this branch locally and tested it
 - [ ] Reviewed this code and left comments
-- [ ] All new functions and methods are commented using plain language
 - [ ] Checked that all code is adequately covered by tests
 - [ ] Made it clear which comments need to be addressed before this work is merged
+- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.
 
-* Code standards are met:
+##### Ensured code standards are met (Code reviewer)
```

Comment:
Should this be #### instead of #####?

Reply:
Oh shoot I merged and didn't scroll up to see this! That wasn't intentional